### PR TITLE
Collections: Label on Node style and field change

### PR DESCRIPTION
**Pull request by minorwm**
# READY FOR REVIEW

# Summary
This removes the block at the top of the article for the collection and puts the information under the date of the article.

# Needed By (Date)
- Sprint end

# Criticality
- Adds new layout for the mag article that is in a collection

# Steps to Test
- Install contrib module: https://www.drupal.org/project/colectomy to take care of colons on the display label
- Switch to this branch  -  7.x-2.x-SOE-2949 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2949)
- `drush cc all`
- `drush fr stanford_soe_helper_collection`
- `drush fr stanford_soe_helper_collection_layouts`
- go to: `admin/structure/types/manage/stanford-magazine-article/display/full`
- Move `This article is part of the series` field right below the `Byline` field. Make sure to link this field to the referenced entity and the label is set to `Inline no colon`.
- Save that display
- See: https://stanfordits.atlassian.net/browse/SOE-2949 for what to see.
- You can test off this page: `magazine/article/craig-criddle-redefining-waste-treatment`

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2949

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_soe_helper_collection/stanford_soe_helper_collection.info [hunk 1]
```diff
@@ -2,7 +2,7 @@ name = Stanford SoE Helper Collection
 description = Adds the collection content type.
 core = 7.x
 package = Stanford
-version = 7.x-2.1-alpha3
+version = 7.x-2.1-alpha4
```
Comment (cjwest) [inline modules/stanford_soe_helper_collection/stanford_soe_helper_collection.info]: Hi @minorwm,

Perhaps things have changed? I thought we didn't assign a new version until it was tagged?
I would expect this to be 7.x-2.1-dev

See https://asconfluence.stanford.edu/confluence/display/SWS/Tagging+a+Product:
How do I create a new tag (release)?, step 12

File: modules/stanford_soe_helper_collection_layouts/stanford_soe_helper_collection_layouts.info [hunk 2]
```diff
@@ -2,7 +2,7 @@ name = Stanford SoE Helper Collection Layouts
 description = Context and other layout items for the collections
 core = 7.x
 package = Stanford
-version = 7.x-2.1-alpha3
```
Comment (cjwest) [inline modules/stanford_soe_helper_collection_layouts/stanford_soe_helper_collection_layouts.info]: Should this be 7.x-2.1-dev?

File: modules/stanford_soe_helper_magazine/stanford_soe_helper_magazine.info [hunk 3]
```diff
@@ -2,7 +2,7 @@ name = Stanford SoE Helper Magazine
 description = Context and other layout items for the magazine articles
 core = 7.x
 package = Stanford
-version = 7.x-2.1-alpha3
```
Comment (cjwest) [inline modules/stanford_soe_helper_magazine/stanford_soe_helper_magazine.info]: 7.x-2.1-dev

File: modules/stanford_soe_helper_collection_layouts/stanford_soe_helper_collection_layouts.info (hunk 2 above)
Comment (minorwm) [reply modules/stanford_soe_helper_collection_layouts/stanford_soe_helper_collection_layouts.info]: fixed

File: modules/stanford_soe_helper_magazine/stanford_soe_helper_magazine.info (hunk 3 above)
Comment (minorwm) [reply modules/stanford_soe_helper_magazine/stanford_soe_helper_magazine.info]: fixed

**Comment by cjwest** [comment]
I'm wondering, @boznik and @minorwm, if this code should go into https://github.com/SU-SWS/stanford_magazine?

**Comment by cjwest** [comment]
Looks right. I'm thinking it could use a little styling love.

screen shot 2018-05-25 at 08 25 51

**Comment by cjwest** [comment]
She looks Gorgeous!

Nice, complete steps to test, @minorwm !

**Pull request merged**